### PR TITLE
Added pl-pl translation

### PR DESCRIPTION
**Pull request by lukaspia**

### CODE REVIEW

File: data/Strings.lua [hunk 1]
```diff
@@ -406,60 +406,60 @@ local Strings = {
...
+        thirst = "Pragnienie",
+        hunger = "Głód",
+        fatigue = "Zmęczenie",
         step = {
             thirst = {
-                "You are hydrated, ",
-                "Low Dehydration, ",
-                "Dehydration, ",
-                "Severe Dehydration, ",
+                "Odpowiednie nawodnienie, ",
```
Comment (jzyinq) [inline data/Strings.lua]: `Jesteś nawodniony` seems to be straightforward translation - same in other cases like:
- `Jesteś najedzony`
- `Jesteś wypoczęty`

File: data/Strings.lua [hunk 2]
```diff
@@ -406,60 +406,60 @@ local Strings = {
...
+                "Zmęczenie, ",
+                "Poważne zmęczenie, ",
             }
         },
         settings = {
             category = {
-                game = "Game Settings",
-                display = "Display Settings"
+                game = "Ustawienia gry",
+                display = "Ustawienia wyświetlania"
```
Comment (jzyinq) [inline data/Strings.lua]: Both words are uppercased in english translation - I would go the same in all translation you've done so `Ustawienia Gry` and so on.

**Review by jzyinq — COMMENTED** [review]
High five from Poland! ;-)

File: data/Strings.lua (hunk 1 above)
Comment (lukaspia) [reply data/Strings.lua]: "Jesteś nawodniony", "Jesteś najedzony", "Jesteś wypoczęty" is more precise translation (1:1), but in Polish it only refers to male gender. I was looking for something more neutral.

File: data/Strings.lua (hunk 2 above)
Comment (lukaspia) [reply data/Strings.lua]: Of course, all settings in CP2077 start with capital letter, but the next word (at least in the Polish version) starts with lower case (unlike English version, where all words are capitalized.). From what I can see, the Italian translation is made similar way. Once, I was translating another game and someone pointed out to me that in Polish, unlike English words, all next words should be in lower case.

File: data/Strings.lua (hunk 1 above)
Comment (jzyinq) [reply data/Strings.lua]: Ok, that's reasonable.

**Comment by FonWasH** [comment]
Thanks guys!

**Pull request merged**